### DISCUSSION TITLE
Which version do I want "two" or "three"?

### DISCUSSION THREAD

**Discussion by scpedicini** [nlp-compromise/nlp_compromise#1110] [Q&A] [answered]
I'm a bit confused by the naming convention of "one"/"two"/"three". 

Compromise **one** seems to indicate that it is a more minimalistic performant library, but the differences between "**two**" and "**three**" are unclear.

- Compromise two calculates the very basic grammar of each word. 
- Compromise three says phrase and sentence tooling.

Does this mean that `compromise/three` does *not* have "basic grammar" of each word? 

Speed is not crucial for me, but absolute accuracy and features are. Does `compromise/three` do everything that `compromise/two` does? 

Thanks!

**Comment by spencermountain** [comment, marked as answer]
Hey Shaun, yep, the three versions are incremental, and build on top of each other. It was meant as a way to offer low-filesize versions to people where this was important, but it may not have been described very clearly. Sounds like you'll want compromise/three, which is he default import - cheers
